frontend/app/dapp/members/components/remove-modal.tsx

```
@@ -83,68 +83,67 @@ const RemoveMemberModal: React.FC<RemoveMemberModalProps> = ({
+        <div className="mb-6 sm:mb-8">
+          <h4 className="text-white text-base sm:text-[18px] font-semibold mb-2 sm:mb-4">
             Assigned Roles
           </h4>
-          <div className="flex gap-3 flex-wrap">
+          <div className="flex gap-2 sm:gap-3 flex-wrap">
             {member.roles.map((role) => (
               <div
-                key={role}
-                className={`flex items-center justify-center text-[14px] px-4 py-2 border-[1px] rounded-full font-medium ${roleColors[role as keyof typeof roleColors]}`}
+                className={`flex items-center justify-center text-xs sm:text-[14px] px-2 sm:px-4 py-1 sm:py-2 border-[1px] rounded-full font-medium ${roleColors[role as keyof typeof roleColors]}`}
               >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)